modules/table/src/main/java/org/apache/ignite/internal/table/distributed/command/CommandUtils.java

```diff
@@ -138,6 +144,88 @@ public static void readRows(byte[] bytes, Consumer<BinaryRow> consumer) {
         }
     }
 
+    /**
+     * Writes a row map to byte array.
+     *
+     * @param map Map a row id to the binary row.
+     * @return Array of bytes.
+     */
+    public static byte[] writeRowMap(Map<RowId, BinaryRow> map) {
+        if (map == null) {
+            return null;
+        }
+
+        try (ByteArrayOutputStream baos = new ByteArrayOutputStream()) {
+            try (ObjectOutputStream oos = new ObjectOutputStream(baos)) {
+                oos.writeInt(map.size());
+
+                for (Map.Entry<RowId, BinaryRow> e : map.entrySet()) {
+                    oos.writeObject(e.getKey());
```

Comment:
Let's use better serialization, something similar to
```
                    oos.writeShort((short) e.getKey().partitionId());
                    oos.writeLong(e.getKey().mostSignificantBits());
                    oos.writeLong(e.getKey().leastSignificantBits());

```

Reply:
Done.